base.Makefile

```diff
@@ -148,10 +156,13 @@ endif
 ifneq (,$(STAGED_PYTHON_FILES))
 	@$(RUFF) format $(STAGED_PYTHON_FILES)
 endif
+ifneq ($(STAGED_JINJA_FILES),)
+	@$(DJLINT) $(STAGED_JINJA_FILES) --reformat || true
```

Comment:
A little bit too specific? It would be more flexible to use specific settings in projects' configuration file (`pyproject.toml`) : https://djlint.com/docs/configuration/#format-css ?

Reply:
Ah oui ok, c'est déjà dans teepy pour le JS donc :+1: